### ISSUE TITLE
Uncaught TypeError: Cannot read property 'onChange' of undefined

### ISSUE THREAD

**Issue by prymitive** [headzoo/react-moment#105]
I've upgraded to 0.9.4 and keep getting errors in the console:

<img width="1595" alt="Screen Shot 2019-10-04 at 23 58 35" src="https://user-images.githubusercontent.com/114288/66244991-f3ef2e00-e702-11e9-831d-556fe1301da6.png">
<img width="291" alt="Screen Shot 2019-10-04 at 23 58 53" src="https://user-images.githubusercontent.com/114288/66244992-f3ef2e00-e702-11e9-965b-7df762dc5d29.png">

Sentry reports:
```
TypeError: Cannot read property 'onChange' of undefined
  at update(../node_modules/react-moment/dist/index.js:1023:21)
  at ? (../node_modules/react-moment/dist/index.js:945:43)
  at Array.forEach(<anonymous>)
  at apply(../node_modules/react-moment/dist/index.js:944:30)
  at r(../node_modules/@sentry/browser/esm/helpers.js:87:17)
```

**Comment by headzoo** [comment]
I published v0.9.5 with a *possible* fix. Can you verify the fix please.

**Comment by prymitive** [comment]
`onChange` error is gone but now I'm getting:

![Screenshot-20191005083418](https://user-images.githubusercontent.com/114288/66251709-fc6b5700-e74a-11e9-92c8-fdb3c16ec54c.png)

**Comment by prymitive** [comment]
I've added a test case to my project that captures this problem, here's an abstracted version:

```
  it("alert timestamp is updated every minute", () => {
    jest.useFakeTimers();
    Moment.startPooledTimer();

    advanceTo(new Date(2018, 7, 14, 18, 36, 41));

    const tree = ComponentUsingReactMoment();
    expect(tree.find("time").text()).toBe("a few seconds ago");

    advanceTo(new Date(2018, 7, 14, 18, 37, 41));
    jest.advanceTimersByTime(61 * 1000);
    expect(tree.find("time").text()).toBe("a minute ago");

    advanceTo(new Date(2018, 7, 14, 19, 36, 41));
    jest.advanceTimersByTime(61 * 1000);
    expect(tree.find("time").text()).toBe("an hour ago");

    advanceTo(new Date(2018, 7, 14, 20, 36, 41));
    jest.advanceTimersByTime(61 * 1000);
    expect(tree.find("time").text()).toBe("2 hours ago");
  });
```

**Comment by headzoo** [comment]
Okay, published v0.9.6. Hopefully fixes the problem.

**Comment by prymitive** [comment]
CI is now green, so looks like it’s fixed, thanks!
